pages/existing_transport_integrations.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Existing server integrations
+weight: 4
+---
+
+### Server integrations
+The spans collected through the instrument API are collected by the zipkin collector and stored in the designated storage.
+The Zipkin server build already integrates http, kafka, rabbitMQ collectors, cassandra, elasticsearch and mysql storage.
+Server integration extends collector or storage.
+
+| Type | Library | Related product | Other notes |
+|:-----|:--------|:----------------|:------------|{% for lib in site.data.transport_integrations %}
+| {{ lib.type }} | {{ lib.library }} | {{lib.product}} | {{ lib.notes }} |{% endfor %}
+{: .wide-table}
+
+
+
+### Zipkin compatible servers
+The following Zipkin compatible servers receive and process zipkin formatted data.
+    
+ - [Skywalking](https://github.com/apache/incubator-skywalking)
```

Comment:
Name should be SkyWalking or Apache SkyWalking. The upper and lower case should be same as we register in Apache

Reply:
I Got it. Of course Apache SkyWalking.